.gitignore

```diff
@@ -35,7 +35,7 @@ localization/.vs
 buildDate.json
 /public/fonts
 /public/electronExtensions.js
-env-config/env-files*.zip
+env-config/*
```

Comment:
I think doing this will not include any of the .js script files we use in our builds. I think we want to keep these .js files in our repo.